packages/flipper-plugin-react-navigation/src/Logs.tsx

```diff
@@ -72,14 +73,12 @@ const Row = styled.button<{ selected: boolean; faded: boolean }>((props) => ({
   'fontSize': theme.monospace.fontSize,
   'textAlign': 'left',
   'padding': `${theme.space.medium}px ${theme.space.large}px`,
-  'color': props.selected ? '#fff' : '#000',
+  'color': props.selected ? '#fff' : '',
   'backgroundColor': props.selected ? theme.primaryColor : 'transparent',
-  'opacity': props.faded ? 0.5 : 1,
```

Comment:
Why is this removed?

Reply:
I am not sure why its showing this line as removed. I pushed again a fix and that line was not changed